client/utils/network.py

```diff
@@ -0,0 +1,42 @@
+"""This module contains utilities for communicating with the ok server."""
+
+from urllib import request, error
+import json
+import time
+import datetime
+import socket
+
+TIMEOUT = 500
+RETRY_LIMIT = 5
+            
+def api_request(access_token, server, route, version, log, 
```

Comment:
The `log` should not be passed in here. If you need to record log messages, you should include the following code at the top:

```
import logging
# other imports

log = logging.getLogger(__name__)
```